More updates to new inspector docs (#11642)
flutter/website: src/content/tools/devtools/inspector.md

Toggling this value enables or disables the hover inspection functionality.

### Enable widget tree auto-refreshing

> The widget tree will automatically refresh after a hot-reload or a navigation event. 

Reviewer: ```suggestion
When enabled, the widget tree automatically refreshes after
a hot-reload or a navigation event. 
```

> Toggling this value enables or disables the auto-refreshing functionality. 

Reviewer: ```suggestion
```
This second sentence doesn't seem necessary, so I suggest removing it.

### Use legacy inspector

> The [legacy inspector][] will be used instead of the new inspector. 
> 
> Toggling this value enables or disables the legacy inspector.

Reviewer: ```suggestion
When enabled, use the [legacy inspector][] instead of the new inspector. 
```

:::note

> Please note that the [legacy inspector][] will be removed in a future release. Let us know if there are issues preventing you from using the new inspector by [filing a bug][].

Reviewer: Line length is too long. Also, don't use "please". Weird, I know.

You don't need to say "please note" because the macro already introduces the note. And we don't say "please": https://developers.google.com/style/word-list#please

```suggestion
The [legacy inspector][] will be removed in a future release.
Let us know if there are issues preventing you from using the new inspector by [filing a bug][].
```
